docs: Update azure READMEs with template contents (#13105)
microsoft/FluidFramework: azure/README.md

See the [package README](./packages/azure-client/README.md) for more information.

> ## @fluidframework/azure-local-services

Reviewer: ```suggestion
## @fluidframework/azure-local-service
```

> The azure-local-services package provides a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `AzureClient` in local mode.

Reviewer: ```suggestion
The azure-local-service package provides a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `AzureClient` in local mode.
```

> See the [package README](./packages/azure-client/README.md) for more information.

Reviewer: ```suggestion
See the [package README](./packages/azure-local-service/README.md) for more information.
```

Author: 🤦‍♂️
